### DISCUSSION TITLE
How do I reset terminal while keep scrollback buffer?

### DISCUSSION THREAD

**Discussion by egolearner** [kovidgoyal/kitty#8164] [Q&A] [answered]
Sometimes, typing character does not work as expected. So we need to reset terminal(by C-l or `reset`)。

It seems that reset terminal also clears scrollback buffer. Is it possible to reset terminal without loosing scrollback buffer?

**Comment by kovidgoyal** [comment, marked as answer]
https://sw.kovidgoyal.net/kitty/conf/#shortcut-kitty.Reset-the-terminal
